src/modules/BootSupportCheck.rb

```diff
@@ -191,6 +194,8 @@ def check_zipl_part
         return false
       end
 
```

Comment:
well, this is protection against expert partitioner known issues from past. I am fine if it will be completely in storage.